content/applications/hr/attendances.rst

```diff
@@ -1,19 +1,474 @@
+- :guilabel:`Add Custom Filter`: Create a custom filter using the pop-up that appears when this is
+  selected.
+
+Groups
+~~~~~~
+
+The default groups that can be selected are:
+
+- :guilabel:`Check In`: This grouping presents a drop-down menu containing the following time period
+  options: :guilabel:`Year`, :guilabel:`Quarter`, :guilabel:`Month`, :guilabel:`Week`, and
+  :guilabel:`Day`. Selected the time period to display all the check-in information, grouped by the
+  selected time period.
+- :guilabel:`Employee`: This group presents the attendance data organized by employee.
+- :guilabel:`Check Out`: This grouping presents a drop-down menu containing the following time
+  period options: :guilabel:`Year`, :guilabel:`Quarter`, :guilabel:`Month`, :guilabel:`Week`, and
+  :guilabel:`Day`. Selected the time period to display all the check-out information, grouped by the
+  selected time period.
+- :guilabel:`Add Custom Group`: This option displays a drop-down menu with a variety of options to
+  group the attendance data by, including :guilabel:`City`, :guilabel:`Country`, :guilabel:`Mode`,
+  and :guilabel:`IP Address`, to name a few.
+
+.. _attendances/errors:
+
+Errors
+------
+
+Entries that contain an error appear on the dashboard in red. If in the :guilabel:`Gantt view`, the
+entry appears with a red background. If in the :guilabel:`List view`, the entry text appears in red.
+
+An error typically occurs when an employee has checked in but has not checked out within the last
+24 hours, or when an employee has a check in and check out period spanning over 16 hours.
+
+To fix the error, the attendance entry must be modified or deleted. Click on the entry to reveal a
+pop-up containing the details for that particular entry. To modify the :guilabel:`Check In` and/or
+:guilabel:`Check Out` information, click on the :guilabel:`Check In` or :guilabel:`Check Out` field
+and a calendar selector appears. Click on the desired date, then use the time selector beneath the
+calendar to select the specific time for the entry. When the information is correct, click
+:guilabel:`Apply.`
+
+.. image:: attendances/errors.png
+   :align: center
+   :alt: The pop-up that allows for modifications to an attendance entry with an error. The calendar
+         selector is shown, and the time selector is highlighted.
+
+When all the information on the pop-up is correct, click :guilabel:`Save & Close`. If the entry no
+longer has an error, the entry now appears in gray instead of red.
+
+To delete an entry, click :guilabel:`Remove` on the pop-up instead of making modifications to the
+entry.
+
+Reporting
+=========
+
+To view attendance reports, click :guilabel:`Reporting` in the top menu. The default report displays
+each employee's attendance information for the past 3 months, in a :guilabel:`Line Chart`.
+
+To default view is a :guilabel:`Graph`. To view the data in a pivot table, click the
```

Comment:
```suggestion
The default view is a :guilabel:`Graph`. To view the data in a pivot table, click the
```

Reply:
I would move this below the section where you explain the different available graph views